docs/guides/cache/index.md

```diff
@@ -1,87 +1,179 @@
 
-- `local` directory cache, that writes the build cache to a local directory on
-  the filesystem.
+- [Local directory cache](./local.md) that writes the build cache to a local
+  directory on the filesystem.
 
-  ([guide](./local.md))
+- [GitHub Actions cache](./gha.md) that uploads the build cache to
+  [GitHub](https://docs.github.com/en/rest/actions/cache) (beta).
 
-- `gha` GitHub Actions cache, that uploads the build cache to [GitHub](https://docs.github.com/en/rest/actions/cache)
-  (experimental).
+- [Amazon S3 cache](./s3.md) that uploads the build cache to an
+  [AWS S3 bucket](https://aws.amazon.com/s3/) (unreleased).
 
-  ([guide](./gha.md))
-
-- `s3` AWS cache, that uploads the build cache to an [AWS S3 bucket](https://aws.amazon.com/s3/)
+- [Azure Blob Storage cache](./azblob.md) that uploads the build cache to
+  [Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/)
   (unreleased).
 
-  ([guide](./s3.md))
-
-- `azblob` Azure cache, that uploads the build cache to [Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/)
-  (unreleased).
+## Command syntax
 
-  ([guide](./azblob.md))
+To use any of the cache backends, you first need to specify it on build with the
+[`--cache-to`](../../reference/buildx_build.md#cache-to) option to export the
+cache to your storage backend of choice. Then, use the
+[`--cache-from`](../../reference/buildx_build.md#cache-from) option to import
+the cache from the storage backend into the current build. Unlike the local
+BuildKit cache (which is always enabled), all of the cache storage backends must
+be explicitly exported to, and explicitly imported from. All cache exporters
+except for the `inline` cache requires that you
+[select an alternative Buildx driver](../drivers/index.md).
 
-To use any of the above backends, you first need to specify it on build with
-the [`--cache-to`](../../reference/buildx_build.md#cache-to) option to export
-the cache to your storage backend of choice, then use the [`--cache-from`](../../reference/buildx_build.md#cache-from)
-option to import the cache from the storage backend into the current build.
-Unlike the local BuildKit cache (which is always enabled), **all** of the cache
-storage backends have to be explicitly exported to and then explicitly imported
-from. Note that all cache exporters except for the `inline` cache, require
-[selecting an alternative Buildx driver](../drivers/index.md).
-
-For example, to perform a cache import and export using the [`registry` cache](./registry.md):
+Example `buildx` command syntax, using import and export cache:
 
 ```console
-$ docker buildx build --push -t <user>/<image> \
-  --cache-to type=registry,ref=<user>/<cache-image> \
-  --cache-from type=registry,ref=<user>/<cache-image> .
+$ docker buildx build . --push -t <user>/<image> \
+  --cache-to type=<cache-type>[,parameters...] \
+  --cache-from type=<cache-type>[,parameters...]
```

Comment:
I really like the idea of using this `[,parameters]` syntax :smile: I think we should probably use that across all the docs?

I think we should keep this as a concrete registry example if we can, I think it goes nicely with the multiple cache example just below.